ftrip.io.booking-service/ftrip.io.booking-service/ReservationRequests/Consumers/ReservationRequestAcceptedEventConsumer.cs

```diff
@@ -0,0 +1,54 @@
+﻿using ftrip.io.booking_service.contracts.ReservationRequests.Events;
+using ftrip.io.booking_service.ReservationRequests.UseCases.DeclineReservationRequests;
+using ftrip.io.booking_service.ReservationRequests.UseCases.ReadReservationRequest;
+using MassTransit;
+using MediatR;
+using Serilog;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ftrip.io.booking_service.ReservationRequests.Consumers
+{
+    public class ReservationRequestAcceptedEventConsumer : IConsumer<ReservationRequestAcceptedEvent>
+    {
+        private readonly IReservationRequestRepository _reservationRequestRepository;
+        private readonly IMediator _mediator;
+        private readonly ILogger _logger;
+
+        public ReservationRequestAcceptedEventConsumer(
+            IReservationRequestRepository reservationRequestRepository,
+            IMediator mediator,
+            ILogger logger)
+        {
+            _reservationRequestRepository = reservationRequestRepository;
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+        public async Task Consume(ConsumeContext<ReservationRequestAcceptedEvent> context)
+        {
+            var requestAcceptedEvent = context.Message;
+
+            var query = new ReadReservationRequestQuery()
+            {
+                AccommodationId = requestAcceptedEvent.AccomodationId,
```

Comment:
Maybe we should add Waiting status filter here?

Reply:
It would be good to add just in case.